package.json

```diff
@@ -2,31 +2,29 @@
   "name": "bobaserver",
   "version": "1.0.0",
   "description": "Server for BobaBoard",
-  "homepage": "https://docs.bobaboard.com/",
+  "homepage": "https://github.com/essential-randomness/bobaserver#readme",
   "bugs": {
-    "url": "https://github.com/BobaBoard/boba-backend/issues"
+    "url": "https://github.com/essential-randomness/bobaserver/issues"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/BobaBoard/boba-backend.git"
+    "url": "git+https://github.com/essential-randomness/bobaserver.git"
   },
   "license": "MIT",
-  "author": "BobaBoard Team",
+  "author": "",
   "main": "dist/server/index.js",
   "scripts": {
-    "build": "tsc && tsc-alias && copyfiles .env .env.prod firebase-sdk.json server/**/*.sql ./dist",
+    "db:start": "docker compose down && docker compose up",
     "deploy": "gcloud config set project bobaboard && gcloud app deploy --version live",
-    "dev:watch": "cross-env DEBUG=bobaserver:*,-*info DEBUG_DEPTH=10 nodemon --watch . --exec node -r tsconfig-paths/register -r ts-node/register server/index.ts -e yaml,ts,sql",
```

Comment:
we should add `yarn run dev` that starts both the server and the db. Double-check, but this should work as a script:

`yarn run db:start & yarn run server:dev`